lib/index.js

```diff
@@ -17,8 +17,9 @@ import {loadConfig, reloadConfig} from './actions/config';
 import HyperContainer from './containers/hyper';
 import configureStore from './store/configure-store';
 
-// Disable pinch zoom
-webFrame.setZoomLevelLimits(1, 1);
+// The default zoom was somehow changed with Electron 3 (or maybe 2). Setting
+// zoom factor to 1.2 brings back the normal default size
+webFrame.setZoomFactor(1.2);
```

Comment:
Unfortunately, it appears zoomed on macOS 😞

Reply:
hmm weird, probably a linux-only thing. I'll investigate

Reply:
Weird, I just tested on MacOS and this line fixes it